common/GlassApi.cs

```diff
@@ -94,6 +94,10 @@
     %digest = GlassAuth.daa.digest(%request);
     %json   = jettisonStringify("object", %digest);
 
+    %request.delete();
```

Comment:
Only `%digest` should be removed here. All else shuld be removed due to jettison recursive removal.